src/ActionsImporter/Commands/Bitbucket/Common.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.CommandLine;
+
+namespace ActionsImporter.Commands.Bitbucket;
+
+public static class Common
+{
+    public static readonly Option<string> InstanceUrl = new("--bitbucket-instance-url")
+    {
+        Description = "The URL of the Bitbucket instance.",
+        IsRequired = false,
+    };
+
+    public static readonly Option<string[]> Namespace = new(new[] { "--namespace", "-n" })
+    {
+        Description = "The Bitbucket namespace(s).",
+        IsRequired = false,
+        AllowMultipleArgumentsPerToken = true
+    };
+
+    public static readonly Option<string> AccessToken = new("--bitbucket-access-token")
+    {
+        Description = "Access token for the Bitbucket instance.",
+        IsRequired = false,
+    };
+
+    public static readonly Option<string> Workspace = new("--workspace")
+    {
+        Description = "The Bitbucket workspace name.",
+        IsRequired = true,
+    };
+
+    public static readonly Option<string> Project = new("--project")
+    {
+        Description = "The Bitbucket project name.",
+        IsRequired = false,
+    };
```

Comment:
```suggestion
    public static readonly Option<string> Project = new("--project-key")
    {
        Description = "The Bitbucket project key.",
        IsRequired = false,
    };
```
I believe this should be `--project-key` for the audit